README.md

```diff
@@ -52,6 +52,16 @@ foo = long_function_name(
     var_two
 )
 ```
+## Yapf - TODO
```

Comment:
More generic section, add information about isort and flake8 to?

Reply:
Smart :+1: Would be nice with a few links etc.

Will look at it tomorrow. Yapf has a few settings (in `setup.cfg`), and I have tried to configure it more or less equally to how the codebase is formatted today (based on pep8). 

It is eg. nice that if you add a comma (`,`) in the end of a list/tuple/set, it will split all elements to separate lines (and the opposite if you don't).